src/Components/Shifting/ShiftDetailsUpdate.tsx

```diff
@@ -166,7 +171,7 @@ export const ShiftDetailsUpdate = (props: patientShiftProps) => {
   const handleSubmit = async (e: any) => {
     const validForm = validateForm();
 
-    if (validForm) {
+    if (!validForm) {
```

Comment:
@patelaryan7751 can you please double check if it should be `!validForm`

Reply:
Yes, before it was not working as expected. So I changed it to make it work as expected.